apps/backend/src/libs/modules/mailer/base-mailer.module.ts

```diff
@@ -0,0 +1,47 @@
+import { type Transporter } from "nodemailer";
+
+import { ErrorMessage } from "~/libs/enums/enums.js";
+import { MailError } from "~/libs/exceptions/exceptions.js";
+import { config } from "~/libs/modules/config/config.js";
+import { HTTPCode } from "~/libs/modules/http/http.js";
+
+class BaseMailer {
```

Comment:
Mail module should be only responsible for sending mail, the email content will be received as arguments